xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -528,6 +528,14 @@ extern "C" {
 
   } ATTRIBUTE_PACKED PVR_TIMER;
 
+  /* PVR_TIMER.iFlags and PVR_RECORDING.iFlags values */
+  const unsigned int PVR_RECORDING_FLAG_UNDEFINED   = 0x00000000; /*!< @brief nothing special to say about this recording */
```

Comment:
If the constants are used for both recordings and timers I would not name them PVR_RECORDING_FLAG_foo. How about just PVR_FLAG_foo?

Reply:
Sorry I meant to remove the PVR_TIMER from the comment. It's only recordings.

Reply:
If they both have the same this would be make sense with same name.

Seen this refer to PVR_TIMER but no iFlags inside.